evaluate2.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#
+#  Copyright 2023 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+if [ -z "$1" ]
+  then
+    echo "Usage: evaluate2.sh <fork name> (<fork name 2> ...)"
+    echo " for each fork, there must be a 'prepare_<fork name>.sh' script and a 'calculate_average_<fork name>.sh' script"
+    echo " there may be an 'additional_build_steps_<fork name>.sh' script too"
```

Comment:
Since we aren't worried about backwards compatibility, should we use this opportunity to start a convention of having the fork scripts in a subdirectory?

Maybe:
```
1brc/
  forks/
    fork1/
      ./prepare.sh
      ./additional_build_steps.sh
      ./calculate_average.sh
    fork2/
      ./prepare.sh
      ./additional_build_steps.sh
      ./calculate_average.sh
```

Thoughts?

Reply:
What about
./fork/build.sh # (optional) use sdk, build native image, docker image
./fork/prepare.sh # (optional) prepare each run via hyperfine --prepare
./fork/run.sh # run.sh # this is what is measured

Reply:
> Since we aren't worried about backwards compatibility,

Ah, I think we should still strive for minimizing the impact as much as possible, in particular considering that there are 50+ pending PRs which would need to be reworked. I agree that the proposed structure is superior, but I'd leave that for another time.